utils/extractor/src/index.ts

```diff
@@ -151,7 +151,7 @@ export const Extractor = (
             status: 'failed',
           })
           await api.jobs.fail(jobId, {
-            info: 'Extraction failed',
+            info: 'files.uploadFile.popovers.extractionFailed',
```

Comment:
```suggestion
            info: 'plugins.extraction.extractionFailed',
```

Reply:
Unless you think I should make another copy of it? I assumed it would be best to leave that language in one place to ensure it stays consistent

Reply:
I would be inclined to copy it over, just on the off chance a customer wants to show a different error message for excel files vs csv

Reply:
moved, thanks!